documentation/en/api-v1-unstable-methods.md

```diff
@@ -3139,6 +3141,113 @@ Inputs:
 
 Response: `"f01234"`
 
+### IpcReadGatewayState
+
+
+Perms: read
+
+Inputs:
+```json
+[
+  "f01234",
+  [
+    {
+      "/": "bafy2bzacea3wsdh6y3a36tb3skempjoxqpuyompjbmfeyf34fi3uy6uue42v4"
+    },
+    {
+      "/": "bafy2bzacebp3shtrn43k7g3unredz7fxn4gj533d3o43tqn2p2ipxxhrvchve"
+    }
+  ]
```

Comment:
Perhaps we should briefly explain these parameters.

Reply:
Unfortunately, this is a generated file automatically created by Lotus when a new method is introduced to the RPC API. It would need deeper changes into Lotus. I will leave it out of scope for now.